src/app/games/epaminondas/tests/AttackEpaminondasMinimax.spec.ts

```diff
@@ -3,48 +3,23 @@ import { Direction } from 'src/app/jscaip/Direction';
 import { Player, PlayerOrNone } from 'src/app/jscaip/Player';
 import { Table } from 'src/app/utils/ArrayUtils';
 import { RulesUtils } from 'src/app/jscaip/tests/RulesUtils.spec';
-import { AttackEpaminondasMinimax } from '../AttackEpaminondasMinimax';
+import { AttackEpaminondasHeuristic, AttackEpaminondasMinimax } from '../AttackEpaminondasMinimax';
 import { EpaminondasMove } from '../EpaminondasMove';
 import { EpaminondasState } from '../EpaminondasState';
-import { EpaminondasNode, EpaminondasRules } from '../EpaminondasRules';
+import { EpaminondasNode } from '../EpaminondasRules';
 import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { AIDepthLimitOptions } from 'src/app/jscaip/MGPNode';
 
-describe('AttackEpaminondasMinimax', () => {
+const _: PlayerOrNone = PlayerOrNone.NONE;
+const O: PlayerOrNone = PlayerOrNone.ZERO;
+const X: PlayerOrNone = PlayerOrNone.ONE;
 
-    let rules: EpaminondasRules;
-    let minimax: AttackEpaminondasMinimax;
-    const _: PlayerOrNone = PlayerOrNone.NONE;
-    const O: PlayerOrNone = PlayerOrNone.ZERO;
-    const X: PlayerOrNone = PlayerOrNone.ONE;
+describe('AttackEpaminondasHeuristic', () => {
+
+    let heuristic: AttackEpaminondasHeuristic;
 
     beforeEach(() => {
-        rules = EpaminondasRules.get();
-        minimax = new AttackEpaminondasMinimax(rules, 'AttackEpaminondasMinimax');
-    });
-    it('should propose 114 moves at first turn', () => {
```

Comment:
where did it go ?

Reply:
c'est un doublon qui appartenait déjà à EpaminondasMinimax, maintenant EpaminondasMoveGenerator.spec.ts

Reply:
Mouais, dans l'optique de tester les comportements j'aurais bien dit qu'on veut que le minimax l'aie aussi (c'est... aussi une unité qu'on veut tester !) mais ouais TPR !